File: test/fixtures/clojure/redirect_command_data.clj
(require '[clj-http.client :as client])
(use '[clojure.java.shell :only [sh]])

(client/post "http://localhost:28139" {:headers {"Content-Type" "application/x-www-form-urlencoded"}
                                       :body (str "foo&" (clojure.java.io/file (sh "bash" "-c" "echo myfile.jg")))})

File: test/fixtures/node/redirect_command_data.js
import fetch from 'node-fetch';
import * as fs from 'fs';
import execSync from 'node:child_process';

fetch('http://localhost:28139', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded'
  },
  body: new ArrayBuffer('foo&', fs.readFileSync(execSync('echo myfile.jg').stdout, 'utf-8'))
});
